skimage/restoration/_nl_means_denoising.pyx

```diff
@@ -10,8 +10,43 @@ ctypedef np.float64_t IMGDTYPE
 
 cdef double DISTANCE_CUTOFF = 5.0
 
-cdef extern from "fast_exp.h":
-    double fast_exp(double y) nogil
+# A fast approximation of the exponential function.
```

Comment:
Shouldn't this be part of an other PR?

Reply:
Ah, sorry, I just noticed that it comes from another PR :smile: !!! It's fine then...